editor/import/resource_importer_texture_atlas.cpp

```diff
@@ -240,17 +248,37 @@ Error ResourceImporterTextureAtlas::import_group_file(const String &p_group_file
 		} else {
 			pack_data.is_mesh = true;
 
+			const int shrink_mask = options["mesh/shrink_mask"];
+			const int grow_mask = options["mesh/grow_mask"];
+			const float simplification = options["mesh/simplification"];
+
 			Ref<BitMap> bit_map;
 			bit_map.instantiate();
 			bit_map->create_from_image_alpha(image);
-			Vector<Vector<Vector2>> polygons = bit_map->clip_opaque_to_polygons(Rect2(0, 0, image->get_width(), image->get_height()));
+
+			const Rect2 rect = Rect2(0, 0, image->get_width(), image->get_height());
+
+			if (shrink_mask > 0) {
+				bit_map->shrink_mask(shrink_mask, rect);
+			}
+
+			if (grow_mask > 0) {
+				bit_map->grow_mask(grow_mask, rect);
+			}
+
+			Vector<Vector<Vector2>> polygons = bit_map->clip_opaque_to_polygons(rect, simplification);
+
+			for (int i = 0; i < polygons.size(); ++i) {
+				polygons.write[i] = Sprite2DEditor::expand(polygons[i], rect, simplification);
```

Comment:
Not sure what's the justification for using `Sprite2DEditor::expand`. I'd guess it hides the mentioned issue with `BitMap::clip_opaque_to_polygons` for some specific test case and thus you've added it. But I'm not sure it's really needed / it fixes anything. And if it would indeed be needed then probably it would make sense to extract this method elsewhere from the `Sprite2DEditor` as `ResourceImporterTextureAtlas` shouldn't be dependant on `Sprite2DEditor`. But again, not sure why / what for it's used.

Reply:
It does not seem to be helpful with correctly generated polygons, removing